src/config.rs

```diff
@@ -145,12 +146,43 @@ impl HTTPUrl {
 #[serde(deny_unknown_fields)]
 pub struct AzureCacheConfig;
 
+fn deserialize_disk_size<'de, D>(deserializer: D) -> std::result::Result<u64, D::Error>
+where
+    D: de::Deserializer<'de>,
+{
+    struct Visitor;
+
+    impl<'de> de::Visitor<'de> for Visitor {
+        type Value = u64;
+
+        fn expecting(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+            formatter.write_str("string disk size with suffix K/M/G/T or integer for Bytes ")
```

Comment:
I'm not sufficiently familiar with `serde`, so just to confirm: is the extra space at the end of this string deliberate?

Reply:
thanks, fixed it